### PR TITLE
jenkins-scripts: add the posibility to run testcases+testplan

### PR DESCRIPTION
**Pull request by roxell**
Signed-off-by: Anders Roxell <anders.roxell@linaro.org>

### CODE REVIEW

**Review by mrchapp — COMMENTED** [review]
I have yet to test this, but isn't $tests undefined?

**Comment by roxell** [comment]
urgh, I should have done tests=${tests:-""} in the beginning

**Comment by mrchapp** [comment]
What's it gonna run with tests=""?

**Comment by roxell** [comment]
oh haha... wait, we should error out if its not set!

File: jenkins-script.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ env
 echo
 [ ! -v GIT_DESCRIBE ] && [ -v LATEST_SHA ] && GIT_DESCRIBE="${LATEST_SHA:0:12}"
 
+tests=${tests:-"lkft-sanity"}
```
Comment (mrchapp) [inline jenkins-script.sh]: Needs "--test-plan".

**Review by mrchapp — CHANGES_REQUESTED** [review]
Also need to update variable to `lava_tests`.

**Review by mrchapp — APPROVED** [review]

**Pull request merged**